packages/playwright-core/src/utils/processLauncher.ts

```diff
@@ -245,7 +245,7 @@ export async function launchProcess(options: LaunchProcessOptions): Promise<Laun
     options.log(`[pid=${spawnedProcess.pid || 'N/A'}] starting temporary directories cleanup`);
     for (const dir of options.tempDirectories) {
       try {
-        rimraf.sync(dir);
+        fs.rmSync(dir, { force: true, recursive: true, maxRetries: 10 });
```

Comment:
I wonder how does the sync version retry? Perhaps we should limit the number of retries here, say to 5, because max total time is quadratic?